modules/administration/pages/iss_v2.adoc

```diff
@@ -1,24 +1,18 @@
 [[iss2]]
-= Inter-Server Synchronization - Version 2
+= Inter-Server Synchronization version 2
```

Comment:
It looks as if you no longer want to use title style capitalization.  If that's the case, we should decide about it and then stick with it ;)

Reply:
We are switching back to sentence-style, in line with SLES documentation.